lm-cli/README.rst

```diff
@@ -0,0 +1,19 @@
+================
+ License Manager CLI
+================
+
+License Manager CLI client
+
+Usage
+-----
+
+...
+
+License
+-------
+* `MIT <LICENSE>`_
+
+
+Copyright
+---------
+* Copyright (c) 2020-2021 OmniVector Solutions <info@omnivector.solutions>
```

Comment:
I'm unsure how we should represent the date here .... `year - year` seems strange. Would it make sense to represent the date with a single year, the current year?

Reply:
hmm .... if we will be publishing these projects independently to pypi then they should each contain the license and a readme

Reply:
> I'm unsure how we should represent the date here .... `year - year` seems strange. Would it make sense to represent the date with a single year, the current year?

This is common, and, it turns out, quite contentious. From my reading just now, it seems like copyright date _should_ at a minimum include the first date year of publication. However, it's not a _requirement_ to specify year at all, since with public repos you can determine the date of first publication.

Since we are using date ranges in lots of our products, maybe we should stick with it here as well, and look at maybe changing it to only the first date of publication in the future (kind of a "fun" busywork task).

Reply:
For now I'm going to add only 2022. We can have this discussion later.